admittance_control_node:
  ros__parameters:
    base_link: "link_0"
    end_effector_link: "link_ee"
    f_ext_th: [2., 2., 2., 0.5, 0.5, 0.5]
    dq_gains: [0.8, 0.8, 0.8, 0.8, 0.8, 0.8, 0.8]
    dx_gains: [4.0, 4.0, 4.0, 40., 40., 40.]
